instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -271,12 +291,27 @@ def process_response(self, request, response):
         if activation and span:
             if is_asgi_request:
                 set_status_code(span, response.status_code)
+                if span.is_recording() and span.kind == SpanKind.SERVER:
+                    # asgi_getter inside asgi_collect_custom_response_attributes
+                    # requires 'headers' key to fetch the actuals headers where
+                    # Django response object has 'headers' properties.
+                    # Thats why I have to create a separate dict with 'headers'
+                    # key to make it compatible with asgi_getter
+
+                    custom_headers = {"headers": response.items()}
```

Comment:
If the key needs to be matched with `AsgiGetter`, we should probably use a constant variable.

Reply:
You mean in place of custom_headers or "headers"?

Reply:
The thing is for asgi framework, "headers" works fine as the the ASGI request and response object has "scope" dict which has the key "headers" by default.
the problem occurs only for django as its request and response object has headers as properties not as dict. so I have used a temporary variable " custom_header" to make the asgi_getter function work consistently.